glow.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: glow
+  version: 2.0.0
+  epoch: 0
+  description: Render markdown on the CLI, with pizzazz!
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+      - gobump
+  environment:
+    CGO_ENABLED: "0"
```

Comment:
This can be removed as go/build handles all these dependencies now:

```suggestion
```

Reply:
Thanks Mark, I should have reviewed the pipeline first. (will do it next time onward)
https://github.com/chainguard-dev/melange/blob/41f30e3eaad19341d38ad2e7a13faf2e501f7d2d/pkg/build/pipelines/go/build.yaml#L3-L7